core/src/util.jl

```diff
@@ -1143,3 +1130,42 @@ function get_demand(user_demand, id, priority_idx, t)::Float64
         demand[id.idx, priority_idx]
     end
 end
+
+"""
+Estimate the minimum reduction factor achieved over the last time step by
```

Comment:
Might be good to mention why we want an underestimation (we don't want to be too strict in clamping the flow right?).